locale/moretrees.es.tr

```diff
@@ -209,3 +209,12 @@ Willow Tree Trunk=Tronco de Árbol de Sauce
 Willow Tree Trunk Slab=Losa de Tronco de Árbol de Sauce
 Willow Tree Trunk Stair=Escalera de Tronco de Árbol de Sauce
 Yellow=Amarillo
+
+# init.lua
+Sapling=
```

Comment:
Retoño
Tablones
Hojas
Valla de Madera
Riel de Valla( de Madera)
Puerta de Valla( de Madera)

Reply:
I'm not sure it is necessary to add 'de madera' to wooden parts. Fences made of special wood or other materials can add that, but for default wood I don't see the need.
In the French and German translations it makes more sense, IMO.